src/sentry/monitors/endpoints/base.py

```diff
@@ -44,15 +44,11 @@ class MonitorEndpoint(Endpoint):
 
     permission_classes = (ProjectMonitorPermission,)
 
-    def convert_args(
-        self,
-        request: Request,
-        organization_slug: str,
-        monitor_slug: str,
-        checkin_id: str | None = None,
-        *args,
-        **kwargs,
-    ):
+    def convert_args(self, request: Request, *args, **kwargs):
+        organization_slug: str = kwargs["organization_slug"]
+        monitor_slug: str = kwargs["monitor_slug"]
+        checkin_id: str | None = kwargs["checkin_id"]
```

Comment:
I don't understand why we'd reduce the signature here, this is less descriptive

Reply:
you can't override the signature of a subclass https://stackoverflow.com/questions/54346721/mypy-argument-of-method-incompatible-with-supertype

Reply:
there's a brand new thing for this: https://typing-extensions.readthedocs.io/en/latest/#override

Reply:
I don't think override helps with this usecase. We have multiple usecases where we _need_ to violate the liskov substitution principle, and this is one of them. Another one is `BaseView.dispatch`